.github/workflows/maven-build.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
-          distribution: 'adopt'
```

Comment:
adopt is no longer maintained https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/